### PR TITLE
[13.0][MIG] web_refresher: Migration to 13.0

### PR DESCRIPTION
**Pull request by joao-p-marques**
@Tecnativa 
TT28809

@pedrobaeza @CarlosRoca13 can you review please?

### CODE REVIEW

**Review by legalsylvain — APPROVED** [review]
LGTM. (Diff review / no test).

**Review by CarlosRoca13 — APPROVED** [review]

File: web_refresher/__init__.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+##############################################################################
+#
+#    Copyright (C) 2019 Compassion CH (http://www.compassion.ch)
```
Comment (chienandalu) [inline web_refresher/__init__.py:3]: The copyright would fit in the __manifest__ better

**Review by chienandalu — APPROVED** [review]
Functional ok :+1: Tiny comment:

**Pull request merged**